samples/bluetooth/peripheral_fast_pair/README.rst

```diff
@@ -247,6 +247,13 @@ Test not discoverable advertising by completing `Testing`_ and the following add
 #. Press **Button 2** to increase the audio volume.
 #. Press **Button 4** to decrease the audio volume.
 
+Battery Notification extension
+------------------------------
+
+To test `Fast Pair Battery Notification extension`_ you have to be paired with at least one Fast Pair Seeker and put Fast Pair Provider in not discoverable advertising mode.
+Currently, Android phones have trouble with the Battery Notification extension and sometimes do not display battery information.
+You can verify that the device is advertising sample battery data using the `nRF Connect for Mobile`_ application.
+
```

Comment:
Shouldn't we add details on how to test the Battery Notification extension. Couldn't find any testing steps here - https://developers.google.com/nearby/fast-pair/specifications/extensions/batterynotification#BatteryNotification

Reply:
Basically these are all details:
1. You have to be paired with at least one Fast Pair Seeker and put Fast Pair Provider in not discoverable advertising mode.
2. You can verify that the device is advertising sample battery data using the `nRF Connect for Mobile`_ application.

Should I split it an put it in points?

Reply:
Yes, that will work. Thanks!